ruby/ql/lib/upgrades/9fdd1d40fd3c3f8f9db8fabf5a353580d14c663a/ruby_scope_resolution_def.ql

```diff
@@ -0,0 +1,16 @@
+class RubyScopeResolution extends @ruby_scope_resolution {
+  RubyScopeResolution() {
+    exists(RubyConstant name | ruby_scope_resolution_def(this, name)) and
+    not ruby_call_def(_, this)
+  }
```

Comment:
I was expecting that we'd simply drop rows where the second column is not a `@ruby_token_constant`, so this charpred seems like it could be deleted.

Reply:
The `not ruby_call_def(_, this)` is still important. It says that things like `expr::Foo()` are not scope resolutions. Even though `Foo` looks like a constant name, it is not.